besu/src/main/java/org/hyperledger/besu/cli/subcommands/RetestethSubCommand.java

```diff
@@ -62,7 +62,7 @@
       paramLabel = "<LOG VERBOSITY LEVEL>",
       description =
           "Logging verbosity levels: OFF, FATAL, ERROR, WARN, INFO, DEBUG, TRACE, ALL (default: ${DEFAULT-VALUE})")
```

Comment:
Yes, you're correct, I'll change it. I let it initially b/c as the field is an actual `org.apache.logging.log4j.Level`, **FATAL** is a possible value, but at the same time it's true that if we are using the slf4j API for logging, we don't have that level available

Reply:
We need to be careful with backwards compatibility here though. If `FATAL` was previously a valid option we likely need to keep it working in some way.  We can provide notice that it's deprecated and will be removed but should follow the proper notice periods for that.

Reply:
That's a good point.
In the current PR there were only 3 calls to `org.apache.logging.log4j.core.Logger#fatal`:
  - [Pruner:168](https://github.com/hyperledger/besu/blob/21.10.9/ethereum/core/src/main/java/org/hyperledger/besu/ethereum/worldstate/Pruner.java#L168)
  - [EvmToolCommand:316](https://github.com/hyperledger/besu/blob/21.10.9/ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/EvmToolCommand.java#L316)
  - [StateTestSubCommand:141](https://github.com/hyperledger/besu/blob/21.10.9/ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/StateTestSubCommand.java#L141)
If i'm understanding this correctly, during a regular client execution, if somebody have set `--logging=FATAL` they would only see a logging value in the case of an irrecoverable situation during a _pruning_ phase before exiting with code `1` and nothing else during the whole lifecycle of the process.
Are these use cases worth going through a deprecation phase?

Reply:
The main issue for me is that if I have `--logging=FATAL` teku shouldn't fail to startup because `FATAL` is an invalid value.  It would probably be reasonable to either disable logging (which is pretty much what would happen anyway) or maybe treat `FATAL` as `ERROR`. Not sure what best reflects the intent there, but definitely shouldn't fail to startup.

Reply:
I think we can work around that, but for having a little more context, why would Teku depend on the CLI option values of Besu?

Reply:
Sorry, wrote teku out of habit but I meant "Besu shouldn't fail to startup because FATAL is an invalid value".

Reply:
In [b440f0a](https://github.com/hyperledger/besu/pull/3285/commits/b440f0a93fd1dd4704d0638e39f7484eba29fdd7) I added an explicit conversion from `FATAL` to `ERROR` with a message about the deprecation. Let me know if you find this behavior better.